### DISCUSSION TITLE
page38

### DISCUSSION THREAD

**Discussion by wjz1316** [rasbt/machine-learning-book#87] [General]
![image](https://user-images.githubusercontent.com/81939467/196025621-9db8ae35-1576-4ca0-b049-71ae0c9b83f2.png)
![image](https://user-images.githubusercontent.com/81939467/196025698-d1ca6da3-6800-48c4-8fb5-8066ff940550.png)
This partial derivative doesn't seem right, the coefficient on L is not included.

**Comment by wjz1316** [comment]
I have found the answer in your errata.

**Reply by rasbt** [reply]
Ah nice, glad it already got resolved and sorry about the inconvenience!
